content/javascript/concepts/arrays/terms/toSorted/toSorted.md

```diff
@@ -0,0 +1,95 @@
+---
+Title: '.toSorted()'
+Description: 'Takes an array and returns a new array with all the elements sorted in ascending order.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Comparison'
+  - 'Functions'
+  - 'Methods'
+  - 'Sort'
+  - 'Unicode'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`toSorted()`** method takes an array and returns a new array with the elements sorted in ascending order. It does not mutate the original array. All undefined elements are sorted to the end of the array.
+
+It does this by converting array elements to [strings](https://www.codecademy.com/resources/docs/javascript/strings) and comparing them by [Unicode code point value](https://developer.mozilla.org/en-US/docs/Glossary/Code_point). 
+
+## Syntax
+
+**Functionless**
```

Comment:
I think it may be preferable to combine all of these in one code block and the text in bold can be added to the right of the syntax as a comment

Reply:
@caupolicandiaz I initially did that but with the code in pseudo and the few lines of additional explanation, it was a little hard to parse. Is that a requirement for you to approve?

Reply:
N/m I'll make it work.

Reply:
✅ in [5a59c46](https://github.com/Codecademy/docs/pull/2716/commits/5a59c46fe45eb19a8b49792b070baee48386a5a4).